### PR TITLE
change user agent

### PR DESCRIPTION
**Pull request by shuisman**
Chrome overwrites the user agent string, which makes the signature fail. Moving to different namespace.

### CODE REVIEW

File: src/resources/resource.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Resource {
     this._results = results || [];
 
     this.signatureFactory.setUrl(this.url);
-    this.signatureFactory.setHeaders({'user-agent': `Usabilla API Node Client/${packageJson.version}`});
+    this.signatureFactory.setHeaders({'x-ub-api-client': `Usabilla API Node Client/${packageJson.version}`});
```
Comment (tzengerink) [inline src/resources/resource.js]: What's the reason you changed this?

Comment (tzengerink) [inline src/resources/resource.js]: Forget it, should read your comments before looking at the code 😄

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "usabilla-api",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (spirosikmd) [inline package.json]: As a good practice better change the version at a different commit. Just before publishing. Please revert this change!

**Comment by shuisman** [comment]
@tzengerink , yhea we just have to track a different parameter?!

**Comment by spirosikmd** [comment]
We should have a test about this though
